worker/lib/index.ts

```diff
@@ -44,6 +45,34 @@ async function setup(): Promise<express.Application> {
 		},
 	};
 
+
+	const supportedTags = [
+		`dut`,
+		`screencapture`,
+		`modem`,
+	]
+	// parse labels and create 'contract'
+	let contract: any = {
+		uuid: process.env.BALENA_DEVICE_UUID,
+	}
+	const balena = getSdk({
+		apiUrl: "https://api.balena-cloud.com/"
+	});	
+	if(typeof process.env.BALENA_API_KEY === 'string' && typeof process.env.BALENA_DEVICE_UUID === 'string'){
+		await balena.auth.loginWithToken(process.env.BALENA_API_KEY);
+		const tags = await balena.models.device.tags.getAllByDevice(process.env.BALENA_DEVICE_UUID);
+		for(let tag of tags){
+			if(supportedTags.includes(tag.tag_key)){
+				if(tag.value === "true")
```

Comment:
We could also assume the presence of a tag with no value to imply true?
```suggestion
				if (!tag.value || tag.value === "true")
```
Need to verify what the sdk returns when a tag has no value, I've made some assumptions here.